Makefile

```
@@ -146,6 +146,14 @@ test-e2e: ## Run the full e2e suite in Docker (builds stack + containerized runn
 		$(E2E_COMPOSE) down -v; \
 		exit $$status
 
+test-e2e-shard: ## Run one CI module shard locally, e.g. make test-e2e-shard PATHS="tests/interview tests/api/interview.api.spec.ts"
```

checkmake (0.3.2)

[warning] Target body for "test-e2e-shard" exceeds allowed length of 5 lines (6).

(maxbodylength)